src/code/z_nmi_buff.c

```diff
@@ -3,6 +3,13 @@
 #define COLD_RESET 0
 #define NMI 1
 
+typedef struct {
+    /* 0x00 */ u32 resetting;
+    /* 0x04 */ u32 resetCount;
+    /* 0x08 */ OSTime duration;
+    /* 0x10 */ OSTime resetTime;
+} NmiBuff; // size >= 0x18
+
 NmiBuff* gNMIBuffer;
```

Comment:
Why `>=`?
Worth noting `gNMIBuffer` is set here: 
`gNMIBuffer = (NmiBuff*)osAppNmiBuffer;`

where `osAppNmiBuffer` is zeroed more than `0x18`:

 `bzero(osAppNmiBuffer, 64);` i.e.  `bzero(osAppNmiBuffer, 0x40);` which is more space than `0x18`.

More just curious if you know